### ISSUE TITLE
Amazon Linux 2 Trouble

### ISSUE THREAD

**Issue by Coding-Badly** [stanislav-tkach/os_info#224]
Platform: Amazon Linux 2
Rust: stable-x86_64-unknown-linux-gnu unchanged - rustc 1.46.0 (04488afe3 2020-08-24)

Running `cargo run --example print_version` produces...

OS information: OracleLinux 2.0.0 [64-bit]
Type: OracleLinux
Version: 2.0.0
Edition: None
Codename: None
Bitness: 64-bit

I think [DISTRIBUTIONS](https://github.com/stanislav-tkach/os_info/blob/master/os_info/src/linux/file_release.rs#L72) and [get_type](https://github.com/stanislav-tkach/os_info/blob/master/os_info/src/linux/file_release.rs#L54) are missing an entry for Amazon Linux 2.

**Comment by Coding-Badly** [comment]
Changing `get_type` to this...

```Rust
fn get_type(name: &str) -> Option<Type> {
    match name.to_lowercase().as_ref() {
       	"amazon linux" => Some(Type::Amazon),
        "arch linux" => Some(Type::Arch),
        "centos linux" => Some(Type::CentOS),
        "ubuntu" => Some(Type::Ubuntu),
        _ => None,
    }
}
```

And adding this to the second element of `DISTRIBUTIONS`...

```Rust
    ReleaseInfo {
        os_type: Type::Amazon,
        path: "/etc/os-release",
        version_matcher: Matcher::KeyValue { key: "VERSION_ID" },
    },
```

I'll get a pull request ready tomorrow.

**Comment by Coding-Badly** [comment]
Nix that.  Just updating `get_type` (and adding a test case) is the right choice.

**Comment by stanislav-tkach** [comment]
Closed with #225.
